install/helm/agones/values.yaml

```diff
@@ -49,6 +49,8 @@ agones:
                 operator: Exists
     generateTLS: true
     safeToEvict: false
+    persistentLogs: true
```

Comment:
Thought: Should this be on by default? I'm thinking most people won't be doing load tests, and stackdriver/fluentd are likely fast enough. WDYT?

Reply:
it's not just load testing, we currently emit enough logs to overwhelm fluentd. For any fleets of significant size there are dropped messages.

Reply:
Ah. In that case, yes - let's keep this on by default. I didn't realise we were getting dropped messages, that's unfortunate.

Sounds like something we should push upstream? (or should we be logging less?)

But otherwise, this makes sense, and looks good to me.

Only other thing I can think of - should we add a section in (to be released in 0.9.0) https://agones.dev/site/docs/guides/troubleshooting/ to let people know that this is an option?

Reply:
sure, I'll submit this one now and make next PR with doc change and ability to use PVC for durable logs